SwiftCompilerSources/Sources/Optimizer/PassManager/ModulePassContext.swift

```diff
@@ -26,6 +26,10 @@ struct ModulePassContext : Context, CustomStringConvertible {
     return String(taking: _bridged.getModuleDescription())
   }
 
+  var enableWMORequiredDiagnostics: Bool {
```

Comment:
Can you put this directly into Context (into `extension Context`)? We might need this in function passes, too.